web/src/backend/jwts.ts

```diff
@@ -89,10 +89,12 @@ export const getUserJWTPayload = (user_id: string, team_id: string) => ({
  * @param team_id
  * @returns
  */
-export const generateUserJWT = async (user_id: string, team_id: string) => {
+export const generateUserJWT = async (
```

Comment:
in a separate PR let's review getting rid of our custom JWT to use the cookie already created by nextjs-auth

Reply:
We making request to hasura with this token

Reply:
let's reevaluate this for something more efficient, maybe keeping the JWT in the session.

Reply:
@pdtfh Token is created for each request and live 1 min. 

[Code](https://github.com/worldcoin/developer-portal/pull/366/files#diff-de1be9e87917d1a5aef88196c32adb4fe4f792125ef9ca64e3a1512c8d93c432R106-R110)

Reply:
yup, meant more around keeping tokens in session to avoid generating them on each request